src/web/wfs/src/main/resources/GeoServerApplication.properties

```
@@ -26,6 +26,10 @@ WFSAdminPage.shapeZipPrjFormat=Use ESRI WKT format for SHAPE\u002DZIP generated
 WFSAdminPage.includeWFSRequestDumpFile=Include WFS request dump file
 WFSAdminPage.storedQueries = Stored Queries
 WFSAdminPage.complexToSimple = Activate complex to simple features conversion
+WFSAdminPage.getFeatureOutputTypes = Allowed Output Format types for a GetFeature request
+OutputTypesFormComponent.outputTypeCheckingEnabled =Enable Output Format type checking
+OutputTypesFormComponent.selectedHeader  =Allowed Output types
+OutputTypesFormComponent.availableHeader =Available Output types
```

Nitpick, just making formatting uniform.

```suggestion
OutputTypesFormComponent.outputTypeCheckingEnabled = Enable Output Format type checking
OutputTypesFormComponent.selectedHeader  = Allowed Output types
OutputTypesFormComponent.availableHeader = Available Output types
```